setup.ps1

```diff
@@ -28,15 +29,22 @@ Disable-ScheduledTasks
 Disable-IPv6To4
 if ($manual_install) {
     Disable-InternetExplorerESC
-    Edit-VisualEffectsRegistry
 }
+Edit-VisualEffectsRegistry
```

Comment:
Not sure if this works without a logged in user.

Reply:
I basically noticed that the registry key edited was not related to any user (HKLM and not HKCU) so I assumed it should not matter, but you may be right..

While testing the changes, I noticed that some effects were disabled (for example, moving a window does not show the window moving real time during the movement, but only an outline of the window) with this approach, but it could be default behavior on Windows Server / Remote Desktop. I don't remember... I'll experiment some more to confirm !

Reply:
That will be cool! Either way it's fine to do it for both manual and automatic.

Reply:
Confirmed : A default install of Windows Server + Remote Desktop does not exhibit this behavior, which means the Edit-VisualEffectsRegistry is properly applied by the script :)

Reply:
Unresolving - I indeed made a mistake on that one, you're right. The Internet Explorer ESC is stored in HKLM, and VisualEffect in HKCU - I got confused between those. Apologies ! 

However, it's weird as  I just rechecked *something* definitely applied. It may have applied the Visual Effect setting to a default account of some sort. Need more investigation.